### ISSUE TITLE
Skylar - Intro to CDO/Netcdf

### ISSUE THREAD

**Issue by kdorheim** [PNNL-PREMIS/SULI2020#11]
During the first part of your summer you'll be using a lot of CDO and R to process NETCDF files. CDO is a  collection of many operators for standard processing of climate and forecast model data. NETCDF is a type of data set that consists of arrays, all of the netcdfs we will be working with have arrays corresponding to horizontal and vertical grids and the time. 

- [ ] Install [CDO](https://code.mpimet.mpg.de/projects/cdo/wiki#Installation-and-Supported-Platforms) we probably will want to go through some examples together but here is some [documentation](https://code.mpimet.mpg.de/projects/cdo/embedded/cdo.pdf) that is a helpful resource. 
- [ ] Download an example CMIP netcdf. We can use one of the [test files](https://github.com/JGCRI/fldgen/blob/master/tests/testthat/data/tas_annual_ipsl-cm5a-lr_rcp8p5_xxx_2006-2099.nc) from fldgen  (a different JGCRI project). 
-  [ ] Download [Panoply](https://www.giss.nasa.gov/tools/panoply/download/) this is a NASA product that visualizes netcdf. 
- [ ] Can you make a map the example data?  What are the dimensions of the data? 

If you want to read about netcdf may be check out the [ncdf4 package documentation ](https://cran.r-project.org/web/packages/ncdf4/ncdf4.pdf) we will use this with R. 

** Please do not feel the pressure to read the CDO or the ncdf4 documentation in detail. I included them because they are useful references to have in your back pocket and might give you an idea of what we will be working with. 

Next steps will be to check out the netcdf in R and also to test out cdo.

**Comment by bpbond** [comment]
A few notes/suggestions-

* Save that CDO user guide to your computer; you'll refer to it frequently (or at least, I do!)
*  Re installing on a Mac, note you can either build from source or use ports or homebrew. I usually just build it, not sure what @kdorheim does
* Finally @MaxFrissell @mlkhorn @10aDing this issue is targeted for Skylar but consider doing bullets 2-4 above (i.e. you don't need CDO, but understanding high-dimensional data files and how to visualize them may be very useful for you all)

**Comment by kdorheim** [comment]
I've installed it with homebrew and I visit the CDO user guide so often I've got the website address partially memorized 🙈

**Comment by skygering** [comment]
@kdorheim Does this look right?
![test_panopoly](https://user-images.githubusercontent.com/60117338/85435581-8313ce80-b53c-11ea-95a2-4d39b7b0d543.png)

The data is graphed with longitude and latitude. The longitude goes from -179.8 to 174.8. The latitude goes from -89.8 to 89.8. I am confused as to how time fits into the graph. I am going to watch some videos on Panopoly before our meeting tomorrow!

**Comment by kdorheim** [comment]
Yup that looks good! So when you have Panopoly it should look something like this...  Do you see the arrows I've circled in red? If you move those arrows up you'll move through time.

<img width="1078" alt="Screen Shot 2020-06-23 at 1 58 34 PM" src="https://user-images.githubusercontent.com/27299759/85438160-b57ff480-b559-11ea-8258-7266c9e13e9d.png">
